pkg/apis/stats/v1alpha1/types.go

```diff
@@ -146,3 +146,43 @@ type RuleTrafficStats struct {
 	Name         string       `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
 	TrafficStats TrafficStats `json:"trafficStats,omitempty" protobuf:"bytes,2,opt,name=trafficStats"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +resourceName=nodeiplatencystats
+// +genclient:onlyVerbs=create
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeIPLatencyStat contains the latency stat of a Node.
+type NodeIPLatencyStat struct {
+	metav1.TypeMeta
+	metav1.ObjectMeta `protobuf:"bytes,1,opt,name=objectMeta"`
+
+	// The list of NodeIPLatency.
+	NodeIPLatencyList []NodeIPLatencyEntry `protobuf:"bytes,2,rep,name=nodeIPLatencyList"`
```

Comment:
I wonder if this should be a map indexed by Node name instead (`map[string] NodeIPLatencyEntry`), with up to 2 `NodeIPLatencyEntry` objects per Node. @tnqn @Dyanngg @heanlan

Reply:
The API conventions recommend list of named objects: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#lists-of-named-subobjects-preferred-over-maps.

When reporting NetworkPolicyStats, I followed the convention and used a list for `RuleTrafficStats`.

Reply:
> I wonder if this should be a map indexed by Node name instead (map[string] NodeIPLatencyEntry), with up to 2 NodeIPLatencyEntry objects per Node. @tnqn @Dyanngg @heanlan

I agree this would be a cleaner approach

Reply:
Based on @tnqn's observation, it seems that a map would not be appropriate.

However, we may still want to group by Node. Please review:
```go
type NodeLatencyStats struct {
	metav1.TypeMeta
	metav1.ObjectMeta

	PeerNodes []PeerNodeLatencyStats
}

type PeerNodeLatencyStats struct {
	NodeName string
	TargetIPs []TargetIPLatencyStats
}

type TargetIPLatencyStats struct {
	TargetIP string
	LastSendTime metav1.Time
	LastRecvTime metav1.Time
	LastMeasuredRTTNanoseconds int64
}
```

I grouped by Node, and made some naming changes as well.

Reply:
The latest proposal looks great to me.

Reply:
That looks good to me, I have updated it.